pepper-apis/src/main/java/org/broadinstitute/ddp/model/user/EnrollmentStatusType.java

```diff
@@ -11,7 +11,7 @@
     ENROLLED(true, false, false,  true),
     EXITED_BEFORE_ENROLLMENT(false, true, true, false),
     EXITED_AFTER_ENROLLMENT(false, true, true, false),
-    CONSENT_SUSPENDED(true, false, true, true);
+    CONSENT_SUSPENDED(true, false, false, true);
```

Comment:
I think the initial idea with `consent_suspended(readonly: true)` was so that it always results in their existing data being read-only until they re-consent. One drawback with changing it to `false` is that read-only is no longer guaranteed and that studies will need to setup configuration to appropriately mark activities read-only.

Reply:
I think we can make this change, but we need to keep in mind to remind studies to add configuration to mark stuff read-only